app/views/replies/_reply.html.erb

```diff
@@ -42,10 +52,19 @@
 
       <% if user_signed_in? %>
         <% if reply.user.id == current_user.id %>
-          <%= link_to "Delete", {:action => 'destroy', :controller => "replies", :id => @reply.id, :new_thread_id => @new_thread.id}, :method => "delete", data: { confirm: 'Are you sure?' }, :class => 'reply-link edit-post'%>
+          <a id="edit-reply-btn-<%= reply.id %>" class="btn btn-default"><i class="fa fa-pencil"></i></a>
+          <%= link_to "<i class='fa fa-trash'></i>".html_safe, {:action => 'destroy', :controller => "replies", :id => @reply.id, :new_thread_id => @new_thread.id}, :method => "delete", data: { confirm: 'Are you sure?' }, :class => 'reply-link edit-post btn btn-default'%>
         <% end %>
       <%end%>
+      <%= link_to "<i class='fa fa-share-alt'></i>".html_safe, new_thread_path + "#reply_" + reply.id.to_s, class: 'btn btn-default' %>
 
     </div>
   </div>
 <% end %>
+
+<script type="text/javascript">
```

Comment:
> Actually this contains erb code in it.

Why can't this just loop over all the replies and use `this.toggle()`. Checkout: http://stackoverflow.com/questions/3204632/jquery-get-a-element-class-based-on-a-prefix#answer-3204714